include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -103,10 +105,8 @@ struct gap
 
     //!\brief Assign from a numeric value (no-op, since gap has only one character).
     //!\param i not used, since gap has only one character
-    constexpr gap & assign_rank([[maybe_unused]] rank_type const i) /*noexcept*/
+    constexpr gap & assign_rank([[maybe_unused]] rank_type const i) noexcept
```

Comment:
This is not in this changes, but you did not remove `gap::GAP`, everywhere else you removed them and also `dna4::UNKNOWN` which I think is handy to have and something that can be represented by.

Or introduce `'-'_gap`

Reply:
> Or introduce '-'_gap

There is a follow-up TODO for this.